src/Symfony/Component/Asset/VersionStrategy/JsonManifestVersionStrategy.php

```diff
@@ -63,6 +63,16 @@ private function getManifestPath(string $path): ?string
             }
         }
 
-        return isset($this->manifestData[$path]) ? $this->manifestData[$path] : null;
+        return $this->manifestData[$path] ?? $this->getPossiblePath($path);
```

Comment:
content of the 2 methods can be inclined here IMHO.

Reply:
👍 yes now seems better